zanata-war/src/main/java/org/zanata/events/ProjectUpdate.java

```diff
@@ -35,4 +35,6 @@
     public static final String EVENT_NAME = "org.zanata.events.ProjectUpdate";
 
     HProject project;
+
+    String oldSlug;
```

Comment:
Yes, that clears that up. Events are still used, just in the hibernate listener now.